src/renderer.h

```diff
@@ -312,7 +313,7 @@ namespace bgfx
 			data.m_parent = _parent;
 			m_hashMap.insert(stl::make_pair(_key, handle) );
 
-			return &m_data[handle].m_value;
+			return std::addressof(m_data[handle].m_value);
```

Comment:
Why not just `&m_data[handle].m_value;`?

Reply:
The Vulkan handle wrapper types overload `operator&` to return a pointer the original Vulkan handle. This makes it incompatible with the declared return type of `Ty*` (e.g. `struct VkImageView*` vs. `::VkImageView*`). `std::addressof` exists for this exact problem, to bypass overloaded address-of operators: https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms/Address_Of